src/test/java/pages/DashboardPage.java

```diff
@@ -25,4 +27,18 @@ public boolean atRequiredPage(){
     public void navigate(){
         open(dashboardPageURL);
     }
+
+    public void clickHelpMenu()
+    {$(By.id("system-help-menu")).shouldBe(visible).click();}
+
+    public void clickJiraCoreHelp()
+    {$(By.id("view_core_help")).shouldBe(visible).click();}
+
+    public void jiraCoreHelpPage(){
+        $("a.cac-header-logo.logo").shouldHave(visible, text("Jira Core Support"));
+    }
+
+    public void dashboardPage(){
```

Comment:
What does this method do?

Reply:
or should I use method 
Assert.assertEquals(dashboardPage.atRequiredPage(), "true");